docs/run-command.md

```diff
@@ -8,12 +8,29 @@ of running scripts in an Azure Linux VM also apply here.
 
 ## Regardless of the Kubernetes control plane status
 
-When executing `run-command` without passing through Kubernetes, we need to provide the information of the node (VMSS instance) where we want to run the given command. To retrieve such information, we can use the [`config import`](./config.md#importing-configuration) command. Once we got it, we can select the node we want to use and all the subsequent `run-command` commands will be executed on that node:
+When executing `run-command` without passing through Kubernetes, we need to provide the information of the node (VMSS instance) where we want to run the given command. To retrieve such information, we can use the [`config import`](./config.md#importing-configuration) command. Once we got it, we can select the node we want to use and all the subsequent `run-command` commands will be executed on that node.
+To try it start by cleaning the current configuration (if any)
 
 ```bash
-# Import the nodes information with the cluster information
-$ kubectl aks config import --subscription mySubID --resource-group myRG --cluster-name myCluster
-$ kubectl aks config show
+kubectl aks config unset-all
+```
+
+Import the nodes information with the cluster information:
```

Comment:
```suggestion
Import the nodes information with the cluster information:

> [!NOTE]
> Passing the cluster information to the `config import` command is optional. However, in this specific case, it will force the command to retrieve the nodes information from the Azure API server instead of the Kubernetes API server, making this approach more resilient to issues on the Kubernetes control plane.
```